docs/electronic_questionnaire_to_data_exchange.rst

```diff
@@ -36,6 +36,12 @@ Schema Definition
     futureproof for new collection instruments.
   survey_id
     The numerical survey number as used across the ONS.
+  case_id
+    The case UUID - used to identify a single instance of a survey collection for a respondent
```

Comment:
Can we keep consistency with punctuation; the other entries are all actual sentences with full stops.